src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const SORT_FIELD_NAME = 'name';
+const SORT_FIELD_LANGTH = 'langth';
```

Comment:
There is a typo in the constant name `SORT_FIELD_LANGTH`. It should be `SORT_FIELD_LENGTH` to correctly represent the sorting field for length.